www/includes/easyparliament/page.php

```diff
@@ -1241,43 +376,10 @@ public function postcode_form() {
         $this->block_end();
     }
 
-    public function member_rss_block($urls) {
-        // Returns the html for a person's rss feeds sidebar block.
-        // Used on MP/Peer page.
-
-        $html = '
-                <div class="block">
-                <h4>RSS feeds</h4>
-                    <div class="blockbody">
-                        <ul>
-';
-        if (isset($urls['appearances'])) {
-            $html .= '<li><a href="' . $urls['appearances'] . '"><img src="' . WEBPATH . 'images/rss.gif" alt="RSS feed" border="0" align="middle"></a> <a href="' . $urls['appearances'] . '">Most recent appearances</a></li>';
-        }
-
-        $HELPURL = new URL('help');
-
-        $html .= '
-                        </ul>
-                        <p><a href="' . $HELPURL->generate() . '#rss" title="An explanation of what RSS feeds are for"><small>What is RSS?</small></a></p>
-                    </div>
-                </div>
-';
-
-        return $html;
-
-    }
-
     public function error_message($message, $fatal = false, $status = 500) {
         // If $fatal is true, we exit the page right here.
         // $message is like the array used in $this->message()
-
-        if (!$this->page_started()) {
-            if (!headers_sent()) {
-                header("HTTP/1.0 $status Internal Server Error");
-            }
-            $this->page_start();
-        }
+        global $page_errors;
```

Comment:
Before, this would mean it sent out a (probably) 500 error if error_message was called, fatal or not, which seems like the right thing to do as we wouldn't want e.g. Google to cache the page and something has gone wrong (even if it manages to show most of the page).